db/migrate/20200404101051_create_text_translations.rb

```diff
@@ -0,0 +1,14 @@
+class CreateTextTranslations < ActiveRecord::Migration[6.0]
+
+  def change
+    create_table :mobility_text_translations do |t|
```

Comment:
this is the option if you use key/value, if you use :json, you could do that in the table itself:

```
    t.json "description"
``